### PR TITLE
Big Dependency Update & Modernization

### PR DESCRIPTION
**Pull request by kekoav**
In an effort to keep this great little project fresh, I'm working on this major version refresh.

A lot of issues are already addressed in upstream projects, and we just need to get those integrated.

## Notable Version Updates

The following are updated along with their implicit dependencies.

* Go (1.11 -> 1.13.10) Go 1.14 waiting for moby/buildkit#1377 first
* Travis Build Version & dev  (xenial -> bionic)
* Alpine images (3.8 -> 3.11)
* opencontainers/runc (latest)
* containerd/containerd (latest)
* moby/buildkit (latest)

## Buildkit Updates

Any other changes I've included were due to API change in buildkit. I've done my best to get this right, but I'm not very familiar with buildkit internals.

## Addresses Issues

Subject to testing, this release should fix the following issues:

* #236

### CODE REVIEW

**Comment by kekoav** [comment]
@AkihiroSuda @jessfraz Can you PTAL?

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,22 @@
 language: go
-dist: xenial
+dist: bionic
 notifications:
   email: true
 go:
-  - 1.11.x
+  - 1.14.x
```
Comment (AkihiroSuda) [inline .travis.yml:6]: 1.14 is untested with the container ecosystem. 1.13 is safer.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM golang:1.11-alpine AS gobuild-base
+FROM golang:1.14-alpine AS img
```
Comment (AkihiroSuda) [inline Dockerfile:1]: 1.13, same as above

File: .travis.yml (hunk 1 above)
Comment (kekoav) [reply .travis.yml:6]: Ok, I can downgrade to 1.13. Where is this information about safety for Go published? How do I track when a Go version is safe in the container ecosystem?

Comment (AkihiroSuda) [reply .travis.yml:6]: https://github.com/moby/buildkit/pull/1377

Comment (kekoav) [reply .travis.yml]: @AkihiroSuda this is now updated.

File: Dockerfile (hunk 2 above)
Comment (kekoav) [reply Dockerfile]: Also updated to 1.13 now.

File: Dockerfile [hunk 3]
```diff
@@ -31,23 +31,18 @@ RUN ./autogen.sh --disable-nls --disable-man --without-audit --without-selinux -
   && make \
   && cp src/newuidmap src/newgidmap /usr/bin
 
-FROM alpine:3.8 AS base
+FROM alpine:3.11 AS base
 MAINTAINER Jessica Frazelle <jess@linux.com>
-RUN apk add --no-cache git
+RUN apk add --no-cache git shadow-uidmap pigz
```
Comment (AkihiroSuda) [inline Dockerfile:36]: `shadow-uidmap` package is unneeded

Comment (kekoav) [reply Dockerfile]: Removed.

**Comment by frezbo** [comment]
@AkihiroSuda Can a new version be released?, it's been a while since the last release.

**Comment by AkihiroSuda** [comment]
cc @jessfraz

**Comment by jessfraz** [comment]
Pushed a new version, thanks @AkihiroSuda for being on top of things been busy with the startup!

FWIW if you ever want to release a new version the commands are:
```
$ make bump-version
$ make tag
$ git push origin ${whatever_tag}
$ git push origin master
```

**Comment by frezbo** [comment]
Thanks @jessfraz, been waiting for a new release for sometime. 👍

**Comment by AkihiroSuda** [comment]
Thanks @jessfraz , but https://r.j3ss.co/repo/img/tags seems not updated?

**Pull request merged**